ikasaneip/error-reporting/src/main/java/org/ikasan/error/reporting/dao/HibernateErrorManagementDao.java

```diff
@@ -87,6 +87,8 @@
     public static final String UPDATE_HARVESTED_QUERY = "update ErrorOccurrenceImpl w set w.harvestedDateTime = :" + NOW + ", w.harvested = 1" +
         " where w.id in(:" + EVENT_IDS + ")";
 
+    private Boolean orderHarvestQuery = false;
```

Comment:
Shouldn't the variable be of type boolean, 

Boolean does not make much sense we cannot have it set to null.

should the variable be called isHarvestQueryOrdered ?

Reply:
> Shouldn't the variable be of type boolean,
> 
> Boolean does not make much sense we cannot have it set to null.
> 
> should the variable be called isHarvestQueryOrdered ?

I'll change to primitive this evening.

Reply:
> Shouldn't the variable be of type boolean,
> 
> Boolean does not make much sense we cannot have it set to null.
> 
> should the variable be called isHarvestQueryOrdered ?

Regarding the variable name, this is a general H2 problem which is the reason why the variable has a general name. We may need to use this flag on other queries moving forward and I do not want an individual variable for each new context.

Reply:
Let me have a think about this. The goal of getting this work done was to have this pull request merged so that it was available for HK as a snapshot for them to develop against for next week when I hands their module back to them. I may make this variable name more general as the H2 problem is general.